src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java

```diff
@@ -1,53 +1,49 @@
 package de.retest.recheck.ignore;
 
-import static de.retest.recheck.RecheckProperties.RETEST_FOLDER_NAME;
-
 import java.io.File;
-import java.io.FileNotFoundException;
-import java.nio.file.NoSuchFileException;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.Optional;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import de.retest.recheck.configuration.ProjectConfigurationUtil;
 import de.retest.recheck.review.GlobalIgnoreApplier;
-import de.retest.recheck.review.counter.NopCounter;
-import de.retest.recheck.review.workers.LoadFilterWorker;
 
+/**
+ * @deprecated Use {@link RecheckIgnoreLocator} instead.
+ */
+// TODO Remove with recheck 2.0
+@Deprecated
```

Comment:
Why deprecate this? This is considered internal API. Just simply remove this and prepare update branches in the other repositories.

Reply:
Since this is non-critical, I would like to leave this as a task for onboarding, thus @Deprecated.